features/support/parameter_types.rb

```diff
@@ -387,7 +382,7 @@ def find_metric(metrics, name)
 
 ParameterType(
   name: 'does',
-  regexp: /|does|does not|doesn't/,
+  regexp: /does|does not|doesn't/,
```

Comment:
was this intentional? I assume originally he idea was to allow a missing `does` to make sentences more natural.

Reply:
Yes but that would mean a double white space and also to remember adding an optional `s` to the verb. I think it's better and more consistent to always use does/doesn't.